rust/aiconfigurator-core/src/operators/gemm.rs

```
@@ -0,0 +1,187 @@
+                latency -= sm_latency;
+            }
+        }
+
+        Ok(PerformanceResult::new(latency, source)
+            .clamp_non_negative()
+            .scaled(self.scale_factor))
+    }
+
+    /// Per-tensor weight count in bytes, matching Python's
+    /// `_weights = n * k * quant_mode.value.memory`.
+    pub fn weights_bytes(&self) -> f64 {
```

Clippy (1.95.0)

[warning] method `weights_bytes` is never used

(warning)